### ISSUE TITLE
YOLOX计算FLOPs有示例demo吗？

### ISSUE THREAD

**Issue by lixiangMindSpore** [Megvii-BaseDetection/YOLOX#894]
![image](https://user-images.githubusercontent.com/78945582/141038959-182a622d-d5cd-41f4-97fd-7ee4e0da7983.png)
运行出错，想看一下YOLOX关于FLOPs上的代码demo，非常感谢

**Comment by FateScript** [comment]
Please refer to pytorch-opcounter.

**Comment by wywywy01** [comment]
使用pytorch-opcounter，出来的计算量会比较少，有一些没有计算进去，请问需要自己配置特定op的计算方式吗？
![image](https://user-images.githubusercontent.com/22450352/147822416-74049cc6-3e9e-4223-83dd-9b170c743bf8.png)


备注：使用mmdet的模型，调用 pytorch-opcounter接口
`from thop import profile
# using pytorch tools
    input = torch.ones(()).new_empty(
                (1, *input_shape),
                dtype=next(model.parameters()).dtype,
                device=next(model.parameters()).device)

    input.cuda()
    flops_torch, params_torch = profile(model, inputs=(input, ))
    print("======  pytorch tools results=========")
    print('flops: {} GFLOPs, params: {} M'.format(flops_torch/1e9, params_torch/1e6))`

**Comment by taoxunqiang** [comment]
使用profile，得到的数值偏少：
model_profile begin!
[INFO] Register count_convNd() for <class 'torch.nn.modules.conv.Conv2d'>.
[INFO] Register count_normalization() for <class 'torch.nn.modules.batchnorm.BatchNorm2d'>.
[INFO] Register zero_ops() for <class 'torch.nn.modules.container.Sequential'>.
[INFO] Register zero_ops() for <class 'torch.nn.modules.pooling.MaxPool2d'>.
[INFO] Register count_upsample() for <class 'torch.nn.modules.upsampling.Upsample'>.
flops: 3.244491744 GFLOPs, params: 5.055855 M
model_profile end!
